cli/args/lockfile.rs

```diff
@@ -84,7 +84,7 @@ pub fn write_lockfile_if_has_changes(
   };
   // do an atomic write to reduce the chance of multiple deno
   // processes corrupting the file
-  atomic_write_file(&lockfile.filename, bytes, cache::CACHE_PERM)
+  atomic_write_file_with_retries(&lockfile.filename, bytes, cache::CACHE_PERM)
```

Comment:
Confirmed this retrying fixed the rare issue I was having in deno_graph's ecosystem tests.